scripts/minikube.sh

```diff
@@ -202,6 +211,14 @@ up)
 
     disable_storage_addons
 
+    #  get kubernetes version we are operating on and accordingly enable feature gates
+    KUBE_MAJOR=$(kube_version 1)
+    KUBE_MINOR=$(kube_version 2)
+    if  [ "${KUBE_MAJOR}" -eq 1 ] && [ "${KUBE_MINOR}" -ge 21 ]
+    then
+    # if kubernetes version is greater than 1.21 enable Generic Ephemeral Volume feature gate
+    K8S_FEATURE_GATES="${K8S_FEATURE_GATES},GenericEphemeralVolume=true"
```

Comment:
as it is beta in 1.21+ already enabled by default. we don't need to enable it explicitly?

Reply:
@Madhu-1 if this script is executed for versions less than 1.21, we dont need to enable this feature gate, thats why tihs check in place.

Reply:
it was added considering the script can be used independently for kube deployments, but may be here, we can skip that and only have e2e check. adjusted the PR..